phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -59,7 +59,9 @@
 								</span>
 								<span class="pm-message-date">
 									<i class="fa fa-clock-o fa-fw" title="{{ message.SENT_TIME }}" aria-label="{{ message.SENT_TIME }}"></i>
-									{{ history.SENT_DATE }}
+									<!--{{ history.SENT_DATE }}-->
```

Comment:
I am not quite sure surrounding Twig with simple HTML will do.
Use {# {{ history.SENT_DATE }} #} instead.

Reply:
Oh, I wasn't trying to change it to twig. I accidentally commented that out so reverted.